frontend/src/components/auth/routes/AdminRoute.tsx

```diff
@@ -0,0 +1,34 @@
+import { userInfo } from "os";
+import React, { useContext } from "react";
+import { Route, Redirect, Switch } from "react-router-dom";
+
+import AuthContext from "../../../contexts/AuthContext";
+import { LOGIN_PAGE, UNAUTHORIZED_PAGE } from "../../../constants/Routes";
```

Comment:
hmm all the ../ is making me think maybe we could try adding folder components (index.ts files) in each component directory to clean up the imports. Article on [folder components](https://medium.com/bootstart/you-should-be-using-folder-components-b30b7d165c39). This is out of the scope for this ticket, but just wanted to bring it up. @Tahmeed27 thoughts on adding a notion ticket for creating folder components?